packages/workbox-build/src/templates/sw.js.tmpl

```diff
@@ -0,0 +1,33 @@
+<% if (importScripts) { %>
+importScripts(<%= importScripts.map(JSON.stringify).join(',') %>);
```

Comment:
I've added support for passing in arbitrary `importScripts: Array<string>`, which I think should help @goldhand.